packages/playwright-config-commons-ui/index.js

```diff
@@ -0,0 +1,106 @@
+// @ts-check
```

Comment:
Where are we getting this config and do we need such all these config changes now?

Reply:
@kilemensi um, that's the default playwright config when you set up playwright...let me remove what we don't need